docs/getting-started.html

```diff
@@ -0,0 +1,40 @@
+<html>
+  <head>
+    <title>
+      Manubot - Docs - Getting Started
+    </title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Quicksand"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,600,700"
+      rel="stylesheet"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Inconsolata"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.8.2/css/all.css"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="../styles.css" />
+  </head>
+  <body>
+    <header style="height: 100%">
+      <video width="100%" controls autoplay muted>
+        <source src="https://ndownloader.figshare.com/files/14874821" type="video/mp4" >
+      </video>
+      <h2>Coming Soon</h2>
+      <i class="fas fa-drafting-compass fa-lg"></i>
+      <br>
+      <div>
+        We are currently in the process of redesigning the Manubot workflow to be simpler and easier to use.
+        We are waiting until this is complete to continue writing the docs, as it will significantly change how Manubot works.
+        In the meantime, please refer the <a href="https://github.com/manubot/rootstock/blob/master/USAGE.md">documentation</a>, <a href="https://github.com/manubot/rootstock/issues">discussions</a>, and <a href="https://github.com/manubot/rootstock">template manuscript</a> on <a href="https://github.com/manubot">GitHub</a>.
```

Comment:
This would be a good place to link to https://github.com/manubot/try-manubot/ to encourage experimentation.

Reply:
Yeah. Great idea.